widgets/src/c3/Gauge.js

```diff
@@ -41,7 +41,8 @@
         this.c3Chart.internal.config.gauge_label_format = this.value_format() === "Percent" ? null : function (value, ratio) { return value; };
         this.c3Chart.internal.config.gauge_label_show = this.show_labels();
         this.c3Chart.load({
-            columns: [[this._columns, this._data]]
+            columns: [[this._columns, this._data]],
+            unload:true
```

Comment:
I am not sure what this is trying to fix?  Plus it does not look correct as per the API - if anything it should be "unload: [this._columns]"?